packages/seacas/libraries/aprepro_lib/apr_builtin.cc

```diff
@@ -72,27 +73,11 @@ namespace SEAMS {
 
 namespace {
   std::unordered_map<size_t, std::vector<std::string>> tokenized_strings;
-  unsigned int hash(const char *symbol)
-  {
-    // Hash function from Aho, Sethi, Ullman "Compilers: Principles,
-    // Techniques, and Tools.  Page 436
-    unsigned int hashval;
-    unsigned int g;
-    for (hashval = 0; *symbol != '\0'; symbol++) {
-      hashval = (hashval << 4) + *symbol;
-      g       = hashval & 0xf0000000;
-      if (g != 0) {
-	hashval = hashval ^ (g >> 24);
-	hashval = hashval ^ g;
-      }
-    }
-    return hashval;
-  }
 
   std::vector<std::string> &get_tokenized_strings(const char *string, const char *delm)
   {
     // key is address of string + hash of delimiter
-    size_t key = hash(string) + hash(delm);
+    size_t key = std::hash<std::string>{}(string) + std::hash<std::string>{}(delm);
```

Comment:
It's more polite to use a hash combiner that promises not to overflow, but I suppose it doesn't matter a whole lot.

Reply:
Unsigned will wrap on overflow, so isn't undefined behavior as it would be with signed overflow.  Not trying to be impolite, but don't see that this is an issue here...

Reply:
I did approve the PR so don't worry :)